python-package/lightgbm/callback.py

```diff
@@ -54,7 +54,23 @@ def _format_eval_result(value: _EvalResultTuple, show_stdv: bool = True) -> str:
         raise ValueError("Wrong metric value")
 
 
-def log_evaluation(period: int = 1, show_stdv: bool = True) -> Callable:
+class _LogEvaluationCallback:
+    """Internal log evaluation callable class."""
+
+    def __init__(self, period: int = 1, show_stdv: bool = True) -> None:
+        self.order = 10
+        self.before_iteration = False
+
```

Comment:
very very minor comment, but could this empty line be removed, to group all assignments to `self.*` properties?

Reply:
I believe it would be better for code readability to split properties that are common for all callbacks (`order` and `before_iteration`) and specific properties for a particular callback. WDYT?
Refer to
https://github.com/microsoft/LightGBM/blob/417c732cc05978276d533c43070d24dfd6c892b7/python-package/lightgbm/callback.py#L204-L210

Reply:
ah I see, that makes sense. Ok yeah I'm fine with that too! I don't feel strongly either way.